Content.Server/Polymorph/Systems/PolymorphSystem.cs

```diff
@@ -232,6 +235,16 @@ private void OnDestruction(Entity<PolymorphedEntityComponent> ent, ref Destructi
             _damageable.SetDamage(child, damageParent, damage);
         }
 
+        // Begin DeltaV - Drop MindContainer entities on polymorph
```

Comment:
make and raise a BeforePolymorphedEvent
then add a system to handle it for containermanager
the hands bit is redundant